R/nhanes_translate.R

```diff
@@ -36,6 +36,10 @@ nhanesTranslate <- function(nh_table, colnames=NULL, data = NULL, nchar = 32,
     message('Column name is required')
     return(NULL)
   }
+
+  if(dxa==FALSE & !is.na(collection_date) & !is.na(container_version)){
```

Comment:
Same comment as above. Not sure how often the DXA tables are accessed since nobody has ever commented about them. Similarly, do you include the National Youth fitness survey in docker? https://www.cdc.gov/nchs/nnyfs/index.htm

Reply:
Hi Chris,  
  I think that they are too large.  When I look at the table reference I see this in the html
<td class="text-center">
--
  | 1999-2006
  | </td><td class="text-center">
  | <a href="/Nchs/Nhanes/Dxa/Dxa.aspx">All Years Doc</a>
  | </td><td class="text-center">
  | <a href="/Nchs/Nhanes/Dxa/Dxa.aspx">All Years Data</a>
  | </td><td class="text-center">
  | Updated December 2016
  | </td>
Which I think means they are in yet another odd location and all lumped together - we may want to just download and store all the URLs from that webpage and index them by the table name - then we would always get the right location for either the data or the docs.

Of note they don't seem to put the limited access data there.

There will be a manifest (soon) of what is in and what isn't for now.  Then we will need to adjust our code to address that.  I believe we can find a way to redirect back to your nhanesA code for anything that is not included.  

And no, nothing on the National Youth fitness survey.  Do you think we should add it?

Reply:
DXA files are much larger, and if they can be retrieved using standard nhanesA then it's probably not worth the effort to add those seldom-used tables to Docker.
 
For the National Youth survey, those tables are in the same format as other NHANES tables. They're not large and were easy to include. There are some "if" statements in several functions to handle the youth survey. FWIW, I have never receive feedback regarding the youth survey so I wouldn't feel compelled to include it in Docker.